cc_proxy.go

```diff
@@ -64,25 +65,7 @@ func (p *ccProxy) connectProxy(proxyURL string) (*api.Client, error) {
 		return nil, err
 	}
 
-	return api.NewClient(conn.(*net.UnixConn)), nil
-}
-
-func (p *ccProxy) allocateProxyInfo() (ProxyInfo, error) {
-	if p.client == nil {
-		return ProxyInfo{}, fmt.Errorf("allocateIOStream: Client is nil, we can't interact with cc-proxy")
-	}
-
-	ioBase, _, err := p.client.AllocateIo(2)
-	if err != nil {
-		return ProxyInfo{}, err
-	}
-
-	proxyInfo := ProxyInfo{
-		StdioID:  ioBase,
-		StderrID: ioBase + 1,
-	}
-
-	return proxyInfo, nil
+	return client.NewClient(conn.(*net.UnixConn)), nil
```

Comment:
Hum, net.UnixConn isn't a requirement for a client anymore as we don't do fd passing in 3.0. Filed https://github.com/clearcontainers/proxy/issues/23 to remind us.